webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidPlural.java

```diff
@@ -82,5 +82,13 @@ public AndroidPluralBuilder setComment(String comment) {
     public AndroidPlural build() {
       return new AndroidPlural(name, comment, items);
     }
+
+    public boolean canBeAdded(AndroidPluralItem pluralItem) {
+      return this.items.stream()
+          .noneMatch(
+              item ->
+                  item.getQuantity().equals(pluralItem.getQuantity())
+                      && !item.getContent().equals(pluralItem.getContent()));
```

Comment:
This was added for cases when there are duplicates with the same content, so the AndroidPluralDuplicateKeyException is thrown

Reply:
Can you explain the logic and why it's used here? Method name should be clearer